build_A_analytical.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+def build_A_analytical(material_property_library, mesh):
+    """ While not intended for long term use, 
+	    this function demonstrates that our steady state model matches
+		the analytical solution.
+		inputs:
+		material_property_library contains relevant material data
+		mesh contains the radial positions of the nodes
+		
+		outputs:
+		T is the array of temperatures at each node.
+    """
+
+    k = material_property_library['k']
+    rho = material_property_library['rho']
+    c = material_property_library['c']
+    N = len(mesh)
+    r_outer = mesh[N-1]
+    g = 1.1*10**10
+    C2 = 800+g*r_outer**2/(6*k)

```

Comment:
800 represents the boundary condition temperature, and should be an input to the function if we decide to continue using this function.